ci-operator/step-registry/ipi/install/rhobs-loki/ipi-install-rhobs-loki-commands.sh

```diff
@@ -0,0 +1,706 @@
+            # long enough to exfiltrate the last set of logs from static pods
+            # from things like etcd and the kube-apiserver. To do that, we need
+            # to stay alive longer than the longest shutdown duration will be
+            # run, which should be 135s from kube-apiserver.
+            exec:
+              command: ["sleep", "150"]
+        name: promtail
+        ports:
+        - containerPort: 3101
+          name: http-metrics
+        readinessProbe:
+          failureThreshold: 5
+          httpGet:
+            path: "/ready"
+            port: http-metrics
+          initialDelaySeconds: 10
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        securityContext:
+          privileged: true
+          readOnlyRootFilesystem: true
+          runAsGroup: 0
+          runAsUser: 0
+        volumeMounts:
+        - mountPath: "/etc/promtail"
+          name: config
+        - mountPath: "/run/promtail"
+          name: run
+        - mountPath: "/var/lib/docker/containers"
+          name: docker
+          readOnly: true
+        - mountPath: "/var/log/pods"
+          name: pods
+          readOnly: true
+        - mountPath: "/var/log/kube-apiserver"
+          name: auditlogs-kube-apiserver
+          readOnly: true
+        - mountPath: "/var/log/openshift-apiserver"
+          name: auditlogs-openshift-apiserver
+          readOnly: true
+        - mountPath: "/var/log/oauth-apiserver"
+          name: auditlogs-oauth-apiserver
+          readOnly: true
+        - mountPath: "/var/log/journal"
+          name: journal
+          readOnly: true
+      - args:
+        - --https-address=:9001
+        - --provider=openshift
+        - --openshift-service-account=loki-promtail
+        - --upstream=http://127.0.0.1:3101
+        - --tls-cert=/etc/tls/private/tls.crt
+        - --tls-key=/etc/tls/private/tls.key
+        - --cookie-secret-file=/etc/tls/cookie-secret/cookie-secret
+        - '--openshift-sar={"resource": "namespaces", "verb": "get"}'
+        - '--openshift-delegate-urls={"/": {"resource": "namespaces", "verb": "get"}}'
+        image: registry.redhat.io/openshift4/ose-oauth-proxy:latest
+        imagePullPolicy: IfNotPresent
+        name: oauth-proxy
+        ports:
+        - containerPort: 9001
+          name: metrics
+          protocol: TCP
+        resources:
+          requests:
+            cpu: 20m
+            memory: 50Mi
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /etc/tls/private
+          name: proxy-tls
+        - mountPath: /etc/tls/cookie-secret
+          name: cookie-secret
+      - args:
+        - --oidc.client-id=\$(CLIENT_ID)
+        - --oidc.client-secret=\$(CLIENT_SECRET)
+        - --oidc.issuer-url=https://sso.redhat.com/auth/realms/redhat-external
+        - --margin=10m
+        - --file=/tmp/shared/prod_bearer_token
```

Comment:
Is this mapped properly? I can't see a secret being mounted to this location.

Reply:
Ah, right, its probably not being mounted in promtail container, fixed